client/src/components/main.scss

```diff
@@ -1,16 +1,21 @@
 .main-page-button {
-  margin-top: 40px;
   margin: auto;
   margin-top: 2em;
   color: white;
   display: block;
   background-color: rgb(20, 213, 104);
   width: 12em;
   height: 4em;
+  line-height: 4em;
   border: none;
   border-radius: 2em;
   text-transform: uppercase;
   font-weight: bold;
+  text-decoration: none;
+  text-align: center;
+  &:hover {
+    color: white;
+  }
```

Comment:
Should this be:
- background color: white
- (text) color: green?

Reply:
This is one of the buttons that has a green background, but for Router, instead of <button>, we use <NavLink>.

Some of the default css styles that would apply to buttons don't apply to this. It has it where when I hovered over the button for 'Register', the text turned to blue, so I added hover text color white.